### PR TITLE
fix: recognize background-image as correct (#42738)

### LINKED ISSUE #42738: [BUG] Throws error when using `background-image` instead of `background` (Learn Applied Visual Design - Add Background Image)
**Affected page**
[Learn Applied Visual Design: Create Texture by Adding a Subtle Pattern as a Background Image](https://www.freecodecamp.org/learn/responsive-web-design/applied-visual-design/create-texture-by-adding-a-subtle-pattern-as-a-background-image)

**Your code**
Copy and paste the code from the editor that you used in between the back-ticks below:

```
<style>
  body {
    background-image: url('https://cdn-media-1.freecodecamp.org/imgr/MJAkxbh.png');
  }
</style>
```

**Expected behavior**
Both `background` and `background-image` should be accepted as correct answers.

**System (please complete the following information):**
 - Device: Laptop / Tablet
 - OS: Both iOS and Windows 10
 - Browser: Firefox

**Additional context**
Reporting this as the ability to accept multiple right answers was apparent with the `margin` and `padding` lessons (example: accepted both clockwise notation and -top, -right, -bottom, -left declarations). The same should be done with a `background` vs `background-image` issue.

### PR DESCRIPTION
**Pull request by yasiralamriki**
The page mentioned in #42738 before would not accept background-image and background, now it should.

Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #42738

### CODE REVIEW

File: curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-texture-by-adding-a-subtle-pattern-as-a-background-image.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Your `body` element should have a `background` property set to a `url()` with th
 ```js
 assert(
   code.match(
-    /background:\s*?url\(\s*("|'|)https:\/\/cdn-media-1\.freecodecamp\.org\/imgr\/MJAkxbh\.png\1\s*\)/gi
+    /background(-image)?:\s*?url\(\s*("|'|)https:\/\/cdn-media-1\.freecodecamp\.org\/imgr\/MJAkxbh\.png\1\s*\)/gi
```
Comment (ieahleen) [inline curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-texture-by-adding-a-subtle-pattern-as-a-background-image.md:25]:
```suggestion
    /background(-image)?:\s*?url\(\s*("|'|)https:\/\/cdn-media-1\.freecodecamp\.org\/imgr\/MJAkxbh\.png\2\s*\)/gi
```

as there is a new capture group this needs to be changed to

Comment (ieahleen) [inline curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-texture-by-adding-a-subtle-pattern-as-a-background-image.md:25]:
```suggestion
    /background(-image)?:\s*?url\(\s*("|'|)https:\/\/cdn-media-1\.freecodecamp\.org\/imgr\/MJAkxbh\.png\2\s*\)/gi
```

as there is a new capture group this needs to be changed too

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]

**Comment by ieahleen** [comment]
Please make sure to run the tests locally before submitting a PR that changes tests
Like shown in https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=testing-challenges

**Comment by ieahleen** [comment]
Hey @ajl223 

We would love to be able to merge your changes but it looks like there is an error with the CI build. ⚠️

Once you resolve these issues, we will be able to review your PR and merge it. 😊

---

Feel free to reference the [contributing guidelines](https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=testing-challenges) for instructions on running the CI build locally. ✅

**Comment by ajl223** [comment]
Yeah sure I will soon sorry for the late reply

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍 

Congratulations on your first PR!

Note that two approving reviews are needed for a PR to be accepted

**Review by Sembauke — APPROVED** [review]

**Pull request merged**